engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFbo.java

```diff
@@ -32,6 +32,9 @@
  *
  * When this state change is reset opengl's default framebuffer (usually the display) is bound again.
  * Similarly, nodes that do not take advantage of this state change will normally write to the default framebuffer.
+ *
+ * Type: Set
+ * Corresponding Reset: UnbindFbo
```

Comment:
Actually, I just noticed you made a number of private classes in line with this comment. Never mind then!